main.nf

```diff
@@ -373,8 +375,12 @@ process bwa {
 
 process samtools {
     tag "${bam.baseName}"
-    publishDir path: { params.saveAlignedIntermediates ? "${params.outdir}/bwa" : params.outdir }, mode: 'copy',
-               saveAs: {filename -> params.saveAlignedIntermediates ? filename : null }
+    publishDir path: "${params.outdir}/bwa", mode: 'copy',
+               saveAs: { filename ->
+                   if (filename.indexOf(".stats.txt") > 0) "stats/$filename"
+                   else if (filename.indexOf(".flagstat.txt") == -1) "flagstats/$filename"
```

Comment:
There is no `flagstat` command in the scripts I guess?

Reply:
ah oops, no I put it in then removed it again